src/modules/locations.ts

```diff
@@ -76,9 +76,9 @@ export function url(path: string = '/', query: Record<string, string> | URLSearc
 
 export default {
   proposals: (options: Partial<ProposalListPage & ProposalListViewFilter & ProposalsStatusFilter & ProposalsTypeFilter & ProposalsModal> | URLSearchParams = {}) => url('/', options),
-  proposal: (proposal: string) => url(`/proposal/`, { id: proposal }),
+  proposal: (proposal: string, newProposal: boolean = false ) => url(`/proposal/`, { id: proposal, newProposal: newProposal.toString()}),
```

Comment:
extra params should be used as options and the shorter version is preferred

```suggestion
  proposal: (proposal: string, options: { new?: boolean } = {} ) => url(`/proposal/`, { id: proposal, ...options }),
```

Reply:
we are going with this definition `options: { new?: "true" } = {}` because the url function def expects strings